app/src/lang/available-languages.yaml

```diff
@@ -1,58 +1,62 @@
 af-ZA: Afrikaans (South Africa)
 ar-SA: Arabic (Saudi Arabia)
+bg-BG: Bulgarian (Bulgaria)
 bn-IN: Bengali (India)
 br-FR: Breton (France)
-bg-BG: Bulgarian (Bulgaria)
 bs-BA: Bosnian (Bosnia and Herzegovina)
 ca-ES: Catalan (Spain)
-zh-CN: Chinese (Simplified)
-zh-TW: Chinese (Traditional)
-hr-HR: Croatian (Croatia)
+ckb-IR: Kurdish Sorani (Iraq)
 cs-CZ: Czech (Czech Republic)
 da-DK: Danish (Denmark)
-nl-NL: Dutch (Netherlands)
+de-DE: German (Germany)
+el-GR: Greek (Greece)
 en-CA: English (Canada)
 en-GB: English (United Kingdom)
 en-US: English (United States)
+eo: Esperanto
```

Comment:
Interesting; so far all of these are "locales" (eg language+region) rather than just "language", but I guess that Esperanto doesn't have any region 🤔

Reply:
Update on this: Crowdin uses `eo-UY`

Reply:
```suggestion
eo-UY: Esperanto
```

Reply:
> Interesting; so far all of these are "locales" (eg language+region) rather than just "language", but I guess that Esperanto doesn't have any region thinking

I think `eo-001` would be a much better choice, since Esperanto is a world language, not just an Uruguayan language. `eo-001` is also used by [Unicode CLDR](https://github.com/unicode-org/cldr/blob/main/common/main/eo_001.xml), and it is also in line with `es-419` that is already used in the file. (`001` and `419` are [UN M49 codes](https://en.wikipedia.org/wiki/UN_M49).)

Reply:
eo-UY is how our translations tool treats it, so while I agree with you personally, we have to match that in order for the translations sync to function 🤷🏻‍♀️